java/src/main/bin/build.sh

```diff
@@ -147,6 +147,9 @@ install_brew_app "docker"
 docker-machine create --driver virtualbox default
```

Comment:
We could also consider removing the docker-machine related lines, as it should be safe to assume that people can install docker the way they wish.  For instance, on my linux computer, I don't use docker-machine at all.  On my mac, I wouldn't want people's scripts possibly messing with my "default" machine.  I'll add the line, but make a note if you'd like me to remove all docker-machine related commands.